util/mkerr.pl

```diff
@@ -455,6 +453,10 @@ sub help
 #ifndef HEADER_${lib}ERR_H
 # define HEADER_${lib}ERR_H
 
+# ifndef HEADER_SYMHACKS_H
```

Comment:
I still wonder why this guard is necessary...

Reply:
> I still wonder why this guard is necessary...

It's an optimization: these 'external' include guards have the advantage that the compiler (resp. preprocessor) does not need to open and read the entire file over and over again when it's included repeatedly. In the modern days where computers have plenty of RAM and efficient file caching, this optimization might not be so important anymore, but still it might speed up compilation.

Unfortunately, there is still no standardized way to tell the compiler that an include file should be included only once, although most of the modern compilers understand [#pragma once](https://en.wikipedia.org/wiki/Pragma_once), which is most often used in conjunction with normal include guards for portability reasons.

Anyway, this question comes too late for this pull request, since it's only a backport of an already merged pr. If you want the external include guards removed, it would have to be done in a separate pull request.